applications/FluidDynamicsApplication/tests/cpp_tests/test_alternative_qs_vms_dem_coupled_quadratic_element.cpp

```diff
@@ -0,0 +1,113 @@
+//    |  /           |
```

Comment:
I'd add this test to current `test_alternative_qs_vms_dem_coupled_element.cpp`. Note that you're testing the very same implementation but with different template instantiation.